src/docs/environment/index.mdx

```diff
@@ -17,36 +21,82 @@ You're going to be working out of this repository for the remainder of the setup
 
 ## System Dependencies
 
-You'll need to first install Xcode cli tools. (If you try running `git` or `make`, you'll get an xcrun error. Run `xcode-select --install` and follow the instructions.
+### Xcode CLI tools
+
+You'll need to first install Xcode CLI tools. Run this command and follow the instructions:
+
+```shell
+xcode-select --install
+```
+
+### Brew
+
+Once you have Xcode installed, install [Homebrew](http://brew.sh), and then the following command to install
+the various system packages as listed in Sentry's `Brewfile`.
+
+```shell
+brew bundle --verbose
+```
+
+### Docker
 
-Once you have Xcode installed, install [Homebrew](http://brew.sh), and then run `brew bundle --verbose` to install the various system packages as listed in sentry's `Brewfile`.
+NOTE: There's an issue with the latest Docker, thus, we need to uninstall what `brew bundle` installed and then downgrade with:
```

Comment:
I've just verified that https://github.com/getsentry/sentry/pull/22250 works on 2.3.0.5 as well as the most recent 2.5.0.1 - we can remove these instructions.

Reply:
Removed.